runelite-api/src/main/java/net/runelite/api/Varbits.java

```diff
@@ -886,4 +886,14 @@ public final class Varbits
 	 * 1 = active
 	 */
 	public static final int SPELLBOOK_SWAP = 3617;
+
+	/**
+	 * The amount of Curse of the Moons stacks received when fighting the Eclipse Moon or Blue Moon.
```

Comment:
Does this not update when fighting the Blood Moon?

Reply:
Regardless of that, can this javadoc be expanded upon? The wiki is not the most helpful for understanding what these values might be, or what they do. This would also help to understand why we change the color for the eclipse moon buff counter when >=18.

Reply:
The varbit does indeed not update when fighting the Blood Moon ([getVarbitValue every GameTick while getting hit by the blood moon](https://youtu.be/aB2cXuqKnmU)).
The color is changed at >= 18 Curse of the Moon stacks when fighting the **Blue** Moon, because the player's joints will lock up at 18 stacks, which causes their next attack to be canceled and 18 stacks to be removed.
I've added this info to the javadoc. Please let me know if you'd like me to expand on it further or if you'd like to see a different change.

Reply:
This looks better. Do we know enough about the eclipse moon's stacks to add a similar description for it?